ecommerce-ui/src/VacayRentalCard.test.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import VacayRentalCard from './VacayRentalCard';
+import AirBnbs from './_data/airbnbs.json';
+import { prototype } from 'events';
+import { isFunction, isBoolean, isArray } from 'util';
+import { equal } from 'assert';
+
+it('shallow renders VacayRentalCard', () => {
+    const wrapper = shallow(
+    <VacayRentalCard 
+        index = {1}
+        listing = {Element}
+        host = {prototype}
+        hasSuperHost = {isFunction}
+        isSuperhost = {isFunction}
+        // I CANT FIGURE THIS OUT!!!!
```

Comment:
Here's the error:

```
    TypeError: Cannot read property 'isSuperhost' of undefined

       6 |         const index = this.props.index;
       7 |         const listing = this.props.listing;
    >  8 |         const hasSuperHost = {display: this.props.listing.host.isSuperhost  ? 'inline-block' : 'none'};
         |                                                                ^
       9 |
      10 |         return(
      11 |             <li className="listing">
```

To test React components, you'll want to pass in some fake data that represents what the component will receive in the actual app. For example, the `listing` prop could receive an object that looks like:

```js
const fakeListing = {
  host: { isSuperhost: true }
};

const wrapper = shallow(
    <VacayRentalCard 
        index = {1}
        listing = {fakeListing}
       ....
   />
);
```

Same goes for all the other props (you won't want to use `isFunction/isBoolean/isArray` from that util library)